src/wireviz/Harness.py

```diff
@@ -96,15 +96,13 @@ def create_graph(self) -> Graph:
                     [html_line_breaks(connector.type),
                      html_line_breaks(connector.subtype),
                      f'{connector.pincount}-pin' if connector.show_pincount else None,
-                     connector.color, '<!-- colorbar -->' if connector.color else None],
+                     connector.color, html_colorbar(connector.color)],
```

Comment:
I am not against this change, but it probably should be a separate PR since it is unrelated to the image issue

Reply:
I agree. I put it here to demonstrate the power of the new functionality.